apiary-metastore-listener/README.md

```diff
@@ -38,7 +38,8 @@ The following shows an example JSON message representing a "CREATE_TABLE" event:
       "protocolVersion": "1.0",
       "eventType": "CREATE_TABLE",
       "dbName": "some_db",
-      "tableName": "some_table"
+      "tableName": "some_table",
+      "sourceMetastoreUris": "thrift://remote_host:9883"
```

Comment:
Hmmm, we don't prefix any of the other values with `source` so this could also just be `metastoreUris` although perhaps having it here is a bit clearer which metastore the message is referring to. I'm fine either way, not sure what everyone else thinks.

Reply:
Not Relevant anymore.